.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "20minutes"
```

Comment:
maybe `"root": true`

Reply:
I wasn't aware of that configuration. Maybe we should add it to all our projects.